### PR TITLE
add requirements file with kafka and python telegram bot

### PR DESCRIPTION
**Pull request by MysterionRise**
Implementation for #1

### CODE REVIEW

**Review by f1nzer — APPROVED** [review]

File: HOW-TO.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### How to set up your working environment
+
+Install Python 3, ensure you have pip, install the _pipenv_:
+
+```python3 -m pip install --user pipenv```
```
Comment (F1nZeR) [inline HOW-TO.md]: We might mark it with corresponding syntax:
```sh
python3 -m pip install --user pipenv
```

File: HOW-TO.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+and then install the requirement in the virtual environment:
+
+```pipenv install -r requirements.txt```
```
Comment (f1nzer) [inline HOW-TO.md:9]: The same as before

File: HOW-TO.md (hunk 1 above)
Comment (f1nzer) [inline HOW-TO.md:5]: We might mark it with corresponding syntax: ```sh

**Review by f1nzer — CHANGES_REQUESTED** [review]

**Comment by f1nzer** [comment]
I think README.md is a better place for such doc.

**Review by IgorPerikov — APPROVED** [review]
LGTM

**Review by f1nzer — APPROVED** [review]

**Pull request merged**